README.md

```diff
@@ -1,3 +1,129 @@
+    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/components?label=npm%40latest'>
+  </a>
+  <a href='https://github.com/webex/components/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
 
-The library for Webex Components.
+**Webex Components** is a set of *React* components following Webex Teams standard styling, aimed at react developers that want to embed the components into their applications.
+
+
+## Table of Contents
+- [Install](#install)
+- [Usage](#usage)
+- [Contributions](#contributions)
+- [License and Support](#license)
+
+<a name='install'></a>
+## Install
+
+```bash
+npm i -S @webex/components @momentum-ui/react
+```
+
+<a name='usage'></a>
+## Usage
+**Webex Components** are using *Cisco style standard* presentational  [momentum ui](https://github.com/momentum-design/momentum-ui) react components. And because of that we need to import all the fonts, icons, images and core *CSS* manually. There are two ways to do this:
+
+### 1. CSS
+1. Copy the '@momentum-ui/core/fonts' directory to the 'fonts' directory for your site.
+2. Copy the '@momentum-ui/core/images' directory to the 'images' directory for your site.
+3. Copy the '@momentum-ui/core/css/momentum-ui.min.css' to your styles/css directory.
+4. In the `<head>` of your HTML reference the location of your momentum-ui.min.css.
+
+    `<link rel='stylesheet' href='path/to/momentum-ui/css/momentum-ui.min.css'>`
+
+***
+### 2. SCSS
+1. Copy the '@momentum-ui/icons/fonts' directory to the 'fonts' directory for your site.
+2. Add an '$brand-font-folder, $icon-font-path, and $images-path' variables to your app's variables.scss file.
+
+    ```scss
+    $brand-font-folder: 'path/to/fonts/directory';
+    $icon-font-path: 'path/to/fonts/directory';
+    $images-path: 'path/to/images/directory';
+    ```
+
+3. Import '@momentum-ui/core/scss/momentum-ui.scss' into your main entry Scss file AFTER the variables.
+
+    `@import '~@momentum-ui/core/scss/momentum-ui';`
+
+4. Compile your Scss using your static compiler or bundler (Webpack, Grunt, Gulp, etc.)
+***
+
+Now that we have the targeted style library imported into our application, we can go ahead and import the components as it shows in the following code:
+
+  ```js
+  import {WebexAvatar} from '@webex/components';
+  import './App.css';
+
+  function App() {
+    return (
+      <div className='App'>
+        <WebexAvatar title='Webex Avatar' />
+      </div>
+    );
+  }
+
+  export default App;
+  ```
+
+*Happy Coding!*
+
+
+<a name='contributions'></a>
+## Contributions
+As much as we truly appreciate your contributions to this projects, there are some rules that we expect all of our contributors to follow.
+
+### Issues
+Please open an [issue](https://github.com/webex/components/issues) and we will get to it in an orderly manner. Please leave as much as information as possible for a better underestanding.
```

Comment:
This is a good start, we should probably create an issue to setup a more explicit issue-handling process.